Gemfile.lock

```diff
@@ -347,10 +353,11 @@ GEM
     websocket-driver (0.7.6)
       websocket-extensions (>= 0.1.0)
     websocket-extensions (0.1.5)
-    zeitwerk (2.6.9)
+    zeitwerk (2.6.12)
 
 PLATFORMS
-  ruby
+  arm64-darwin-23
```

Comment:
This is likely to cause a proliferation of other developer-specific platforms here and seems an unnecessary/unintended change?  `ruby` is sufficient and what we have in every other repo.